easybuild/easyconfigs/s/scikit-image/scikit-image-0.15.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-image'
+version = '0.15.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
```

Comment:
should be `https://scikit-image.org/` (it's wrong in existing easyconfig too, we should fix that in a separate PR)

Reply:
fixed for existing easyconfigs in #8916

Reply:
done